auth0/src/test/java/com/auth0/android/provider/LogoutManagerTest.java

```diff
@@ -0,0 +1,81 @@
+package com.auth0.android.provider;
+
+import com.auth0.android.Auth0;
+import com.auth0.android.Auth0Exception;
+import com.auth0.android.callback.BaseCallback;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+
+@RunWith(RobolectricTestRunner.class)
+@Config(sdk = 18)
+public class LogoutManagerTest {
+
+    @Rule
+    public ExpectedException exception = ExpectedException.none();
+
+    @Mock
+    Auth0 account;
+    @Mock
+    BaseCallback<Void, Auth0Exception> callback;
+
+    @Before
+    public void setUp() throws Exception {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void shouldNotHaveCustomTabsOptionsByDefault() throws Exception {
+        LogoutManager manager = new LogoutManager(account, callback, "https://auth0.com/callback");
```

Comment:
I wouldn't use `auth0.com` URLs for anything that would be app-configured.

Reply:
This is just a test case and that's some valid url string. Should I use something different?

Reply:
In the spirit of "tests as documentation," I would have this URL be something similar to what a developer would use. In this case, they would not be using an `auth0.com` domain (or, typically, an `http/s` scheme).